extensions/rich_text_components/Image/directives/image_directive.html

```diff
@@ -1,5 +1,10 @@
 <figure>
-  <img ng-src="<[imageUrl]>" alt="<[imageAltText]>"
-       style="display: block; margin-left: auto; margin-right: auto;">
-  <figcaption style="text-align:center;"><[imageCaption]></figcaption>
+  <div ng-if="isLoadingIndicatorShown">
+    <img ng-src="<[imageUrl]>" alt="<[imageAltText]>"
+         style="display: block; margin-left: auto; margin-right: auto;">
```

Comment:
You might also use the `oppia-image-loading-indicator-container` class here (just rename it to just `oppia-image-container`)

Reply:
Added a general class ( center-image-horizontally) to center the image.